internal/telemetry/cloteleminfo.go

```diff
@@ -147,6 +171,34 @@ func UpdateCLFMetrics() error {
 
 	return nil
 }
+
+func ResetCLFMetrics() error {
+
+	CLFInfo := Data.CLFInfo.M
+	CLFInputType := Data.CLFInputType.M
+	CLFOutputType := Data.CLFOutputType.M
+
+	mCLFInfo.With(prometheus.Labels{
+		"healthStatus": CLFInfo["healthStatus"],
+		"pipelineInfo": CLFInfo["pipelineInfo"]}).Set(0)
+
+	mCLFInputType.With(prometheus.Labels{
+		"application":    CLFInputType["application"],
+		"audit":          CLFInputType["audit"],
+		"infrastructure": CLFInputType["infrastructure"]}).Set(0)
+
+	mCLFOutputType.With(prometheus.Labels{
+		"default":        CLFOutputType["default"],
```

Comment:
these all have constants defined in the API code.  we should use them

Reply:
@jcantrill "infrastructure", "application", "audit" are the labels for this metrics. Similarly all supported output types also are the labels applied to other info metric. From readability point of view it is better to leave them as it is. They are not any other variables values here.  Please let me know if you still feel that we should replace them by the constants defined for API code or input types. thanks.

Reply:
> From readability point of view it is better to leave them as it is
I would disagree.  Maintainability is more important to me.  It is common practice in any kind of coding to:
* refactor strings into constants
* reuse constants which provide values across the code base so they can all change as needed when requirements change